### ISSUE TITLE
Change the image registry, repo and version of csi related pods

### ISSUE THREAD

**Issue by wlawton** [kubernetes/kops#14939]
/kind feature

We are keen on sourcing all images used in our K8S cluster from ECR in our own AWS account. We've achieved this except for the csi related pods whose images are listed below. I believe these are deployed to the cluster via KOPS. We are using KOPS version 1.22.2 which has AWS EBS CSI driver enabled by default, so we dont have any csi specific configuration in our KOPS config file. Wondering if it is possible to still have the csi pods deployed and managed via KOPS but to be able to change the image registry, repo and version so we can source images from our own ECR.

k8s.gcr.io/provider-aws/aws-ebs-csi-driver:v1.2.1
k8s.gcr.io/sig-storage/csi-attacher:v3.2.0
k8s.gcr.io/sig-storage/csi-node-driver-registrar:v2.1.0
k8s.gcr.io/sig-storage/csi-provisioner:v2.2.0
k8s.gcr.io/sig-storage/csi-resizer:v1.1.0
k8s.gcr.io/sig-storage/csi-snapshotter:v4.0.0
k8s.gcr.io/sig-storage/livenessprobe:v2.2.0

**Comment by hakman** [comment]
Would https://kops.sigs.k8s.io/operations/asset-repository/ help?

**Comment by wlawton** [comment]
Thanks for the input @hakman. Yes we already have the assets fileRepository and containerProxy configured as part of this exercise. However, since I logged this issue, I have discovered something which has given me pause for thought. The pods using those csi related images I previously referred to are only deployed in our development environment and not test, stage or prod despite having the same version of Kubernetes and KOPS in play in all environments and KOPS config file being in sync between them, so at the moment i'm at a bit of a loss as to why our dev environment has them deployed. Sure enough the dev KOPS config store contains addons/aws-ebs-csi-driver.addons.k8s.io/k8s-1.17.yaml, but its absent elsewhere. All I can think is that some time ago I made an aborted attempt to upgrade Kubernetes in Dev from v1.21 to v1.22, and those csi pods are a side effect of this. I think i need to get to the bottom of this before pursuing this issue further in this forum. Thanks again!
